pkg/apis/feature/flag_names.go

```diff
@@ -23,4 +23,5 @@ const (
 	KReferenceMapping   = "kreference-mapping"
 	NewTriggerFilters   = "new-trigger-filters"
 	TransportEncryption = "transport-encryption"
+	Istio               = "istio"
```

Comment:
I wouldn't add it here but only in a test-specific package

Reply:
Ok. Updated